src/tests/topp/IDFileConverter_6_output.idXML

```diff
@@ -3,8 +3,8 @@
 <IdXML version="1.3" xsi:noNamespaceSchemaLocation="http://open-ms.sourceforge.net/SCHEMAS/IdXML_1_3.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
 	<SearchParameters id="SP_0" db="/nfs/mascot/mascot-2.4/sequence/UPEcoli_SpikeIn/current/UPEcoli_SpikeIn_Jun13.fasta" db_version="" taxonomy="" mass_type="monoisotopic" charges="" enzyme="trypsin" missed_cleavages="0" precursor_peak_tolerance="0" precursor_peak_tolerance_ppm="false" peak_mass_tolerance="0" peak_mass_tolerance_ppm="false" >
 		<FixedModification name="Carbamidomethyl (C)" />
-		<VariableModification name="Deamidated (NQ)" />
-		<VariableModification name="Deamidated (NQ)" />
+		<VariableModification name="Deamidated (N)" />
+		<VariableModification name="Deamidated (Q)" />
```

Comment:
I guess because the descriptions in the modifications db changed, if they ever reflected that. @timosachsenberg do you know if something changed there?
Anyways, as the input is 
```
<aminoacid_modification aminoacid="N" massdiff="+0.98402" mass="115.026943" variable="Y" binary="N" description="Deamidated (NQ)"/>
<aminoacid_modification aminoacid="Q" massdiff="+0.98402" mass="129.042594" variable="Y" binary="N" description="Deamidated (NQ)"/>
```
this seems just right.